### PR TITLE
fix(skia): not activating the window shouldn't prevent the application from loading

### LINKED ISSUE #16881: [X11] The window flashes in Application Startup
### Current behavior

The main window will flashes to Transparent and white (default window background) in Application Startup on X11 platform.

### Expected behavior

The main window can show the final background when Activated.

### How to reproduce it (as minimally and precisely as possible)

1. Create an empty UNO application
2. Running the application on X11 platform in Linux

### Workaround

_No response_

### Works on UWP/WinUI

No

### Environment

_No response_

### NuGet package version(s)

```json
{
  // To update the version of Uno please update the version of the Uno.Sdk here. See https://aka.platform.uno/upgrade-uno-packages for more information.
  "msbuild-sdks":
   {
    "Uno.Sdk": "5.2.139"
  }
}

```

### Affected platforms

Skia (Linux X11)

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

It can not fixed this issues by setting UseOpenGLOnX11 to false:

```csharp
FeatureConfiguration.Rendering.UseOpenGLOnX11 = false;
```

And I think the reason is the UNO create the X11 windows first and render delay. I add the `Console.WriteLine($"Create X11 Window {Stopwatch.GetTimestamp()} Frequency={Stopwatch.Frequency}");` in  `X11XamlRootHost.Initialize`. And then I add the code in `X11SoftwareRenderer.InvalidateRender` :

```csharp
			if (renderCount == 0)
			{
				Console.WriteLine($"First render {Stopwatch.GetTimestamp()}");
				renderCount++;
			}
```

Then I run the application and get the output : 

```
Create X11 Window  276775049460711 Frequency=1000000000
First render 276775252808388
```

It means that the interval between `Create X11 Window` and `First render` is (276775252808388-276775049460711)/1000000000*1000=203.35  milliseconds

### PR DESCRIPTION
**Pull request by ramezgerges**
GitHub Issue (If applicable): closes #16881

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.MacOS/MacOSWindowNative.cs [hunk 1]
```diff
@@ -15,19 +15,21 @@ internal class MacOSWindowNative
 	private readonly WinUIWindow _winUIWindow;
 	private readonly ApplicationView _applicationView;
 
-	public MacOSWindowNative(WinUIWindow winUIWindow, Microsoft.UI.Xaml.XamlRoot xamlRoot)
+	public MacOSWindowNative(WinUIWindow winUIWindow, Microsoft.UI.Xaml.XamlRoot xamlRoot, out Size initialSize)
```
Comment (jeromelaban) [inline src/Uno.UI.Runtime.Skia.MacOS/MacOSWindowNative.cs:18]: We'll need to validate this change on macOS before merging.

File: src/Uno.UI.Runtime.Skia.Wpf/Rendering/OpenGLWpfRenderer.cs [hunk 2]
```diff
@@ -237,8 +238,16 @@ public void Render(DrawingContext drawingContext)
 
 			drawingContext.DrawImage(_backBuffer, new Rect(0, 0, _hostControl.ActualWidth, _hostControl.ActualHeight));
 		}
+
+		if (asd == 0)
+		{
+			asd++;
+			Console.WriteLine($"ramooz first render {Stopwatch.GetTimestamp() / TimeSpan.TicksPerMillisecond}");
+		}
```
Comment (jeromelaban) [inline src/Uno.UI.Runtime.Skia.Wpf/Rendering/OpenGLWpfRenderer.cs:246]: This looks like it can be removed ;)

**Review by jeromelaban — APPROVED** [review]

File: src/Uno.UI.Runtime.Skia.Wpf/Rendering/OpenGLWpfRenderer.cs (hunk 2 above)
Comment (ramezgerges) [reply src/Uno.UI.Runtime.Skia.Wpf/Rendering/OpenGLWpfRenderer.cs:246]: Oops :)

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.3

File: src/Uno.UI.Runtime.Skia.MacOS/MacOSWindowNative.cs (hunk 1 above)
Comment (jeromelaban) [reply src/Uno.UI.Runtime.Skia.MacOS/MacOSWindowNative.cs:18]: This is working properly on macOS.

**Pull request merged**